javaws-wrapper/src/main/java/com/teamscale/jacoco/javaws/WindowsInstallation.kt

```diff
@@ -45,7 +43,7 @@ constructor(workingDirectory: Path) {
     @Throws(InstallationException::class)
     private fun validate() {
         try {
-            FileSystemUtils.mkdirs(backupPaths.backupDirectory.toFile())
+            backupPaths.backupDirectory.toFile().mkdirs()
         } catch (e: IOException) {
```

Comment:
mkdirs does not throw IO exceptions, it returns true or false. please handle accordingly